internal/controller/nimservice_controller.go

```diff
@@ -189,6 +199,11 @@ func (r *NIMServiceReconciler) GetEventRecorder() record.EventRecorder {
 	return r.recorder
 }
 
+// GetK8sType returns the container platform type
+func (r *NIMServiceReconciler) GetK8sType() k8sutil.OrchestratorType {
```

Comment:
nit: call it orchestratorType everywhere?

Reply:
sure